sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventContactCountMethod.java

```diff
@@ -0,0 +1,6 @@
+package de.symeda.sormas.ui.events;
+
+public enum EventContactCountMethod {
+	COUNT_ALL_CONTACTS,
```

Comment:
I'd suggest to use shorter names here:

ALL,
SOURCE_CASE_IN_EVENT

since the enum name already suggests that it's about contact counts